src/views/address/address.html

```diff
@@ -1,67 +1,45 @@
 {% extends "form-template.njk" %}
-{% from "govuk/components/input/macro.njk" import govukInput %}
-{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "hmpo-form/macro.njk" import hmpoForm %}
+{% from "hmpo-submit/macro.njk" import hmpoSubmit %}
+{% from "hmpo-text/macro.njk" import hmpoText %}
+{% set hmpoPageKey = "address-form" %}
+{% set hmpoPageContent = "address-form" %}
 
-{% set hmpoPageKey = "identityAddress" %}
+{% set hmpoPageKey = "address-form" %}
 
 
-{% block innerContent %}
+{% block content %}
 
-  {% call govukFieldset({
-    legend: {
-      text: "What is your address?",
-      classes: "govuk-fieldset__legend--l",
-      isPageHeading: true
-    }
-  }) %}
+  <h1>{{translate("pages.address-form.title")}}</h1>
 
-    {{ govukInput({
-      label: {
-        html: 'Building and street <span class="govuk-visually-hidden">line 1 of 2</span>'
-      },
-      id: "address-line-1",
-      name: "address-line-1",
-      autocomplete: "address-line1"
-    }) }}
+  {{hmpoHtml(translate("links.changePostcode"))}}
```

Comment:
We've got a slightly different form structure, so we're building it up more manually 

:100: